kuma/static/styles/components/wiki/crumbs.scss

```diff
@@ -29,7 +42,18 @@ wiki article bread crumbs
         /* borders increase hit area for mobile without messing with position of arrow */
         border-top: 15px solid transparent;
         border-bottom: 15px solid transparent;
-        line-height: 1;
+        line-height: 1.5;
+        word-break: break-word;
+
+        @media #{$mq-tablet-and-up} {
+            border-top: none;
+            border-bottom: $crumb-vertical-spacing-tablet solid transparent;
+        }
+
+        @media #{$mq-small-desktop-and-up} {
+            border-bottom: $crumb-vertical-spacing-desktop solid transparent;
+            line-height: 1;
+        }
 
         &:after {
             margin: 10px 5px 3px;
```

Comment:
I suspect this is the line causing the misalignment. A bottom margin of 5px looks good to me in tablet mode, though that messes things up at desktop size.

(This would be easier if we could just use text instead of an image here, like https://www.fileformat.info/info/unicode/char/203a/index.htm used with css content. Thought I suppose that having a directional character makes bidi handling harder...)